index.html

```diff
@@ -982,7 +982,17 @@ <h3>Contexts</h3>
 the context.
           </dd>
         </dl>
-
+        <p class="note">
+Though this specification requires that a <code>@context</code> <a>property</a>
+be present, there is no requirement that the value of the <code>@context</code>
+property be processed using JSON-LD. This is to support compatibility with
+certain JSON processing libraries such as those that may be used when the
+<a>verifiable credential</a> is encoded as a JWT.
+For JSON-LD aware processors to properly function, the <code>@context</code>
+needs to be processed. For JSON processors, it is sufficient to ensure that the
```

Comment:
What worries me about this is that if I get sent a JSON-LD document every time, why wouldn't I just always do the (simpler) JSON processing?

Reply:
The only time you need to do any more than that simple processing is if you want to translate the data to use some other `@context` or into some other structure or format. It may be better to say "JSON-LD aware processors can process the `@context` using a full JSON-LD processor as expected. For other processors, the only processing necessary is to ensure that the...".

Reply:
changes made in [665e076](https://github.com/w3c/vc-data-model/pull/535/commits/665e07665cfb3bdfa4c5b625257e49bc709109fa)

Reply:
> The only time you need to do any more than that simple processing is if you want to translate the data to use some other @context or into some other structure or format.

Also, if you're using LD Proofs / LD Signatures to generate a signature (or verify one), then you will have to do C14N, which will require you to do JSON-LD processing. So, if you live in JWT land and don't want to use JSON-LD, you don't have to (modulo the simple `@context` array string comparison). If you want what JSON-LD provides, or want LD Proofs/Signatures, then you can do extra processing to get those benefits.